Public/Sdk/SelfHost/BuildXL/Tools/QTest/Tool.QTestRunner.dsc

```diff
@@ -170,6 +170,7 @@ export function runQTest(args: QTestArguments): Result {
         Cmd.flag("--enableVsJitDebugger", Environment.hasVariable("[Sdk.BuildXL]enableVsJitDebugger")),
         Cmd.flag("--qTestIgnoreQTestSkip", args.qTestIgnoreQTestSkip),
         Cmd.option("--qTestAdditionalOptions ", args.qTestAdditionalOptions, args.qTestAdditionalOptions ? true : false),
+        Cmd.option("--envVarsToKeep ", args.envVarsToKeep || ""),

```

Comment:
>|| "" [](start = 58, length = 5)

You dont need this part. If args.envVarsToKeep is null, BuildXL ignores adding this argument. You can validate it though.